### ISSUE TITLE
Rich text editor a11y improvements

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#5174]
The rich text editor was flagged as not having its label correctly associated with the input by the [Axe Developer Hub](https://github.com/bloom-housing/bloom/issues/5139) test suite.

In addition to associating the label, brief research shows the menu bar should be navigable with the arrow keys as well.

QA Notes

In [core dev](https://partners.bloom.exygy.dev/sign-in), when using your keyboard tabbing through, then when you land on the first menu item (bold) in the rich text editor (which appears on the what to expect field) that you can use your left and right arrow keys to navigate through each menu bar item, which should wrap around when you get to the end.
